exercises/leap/canonical-data.json

```diff
@@ -1,39 +1,29 @@
 {
    "cases": [
       {
-         "description": "leap year in twentieth century",
-         "input": 1996,
-         "expected": true
-      },
-      {
-         "description": "odd standard year in twentieth century",
-         "input": 1997,
-         "expected": false
-      },
-      {
-         "description": "even standard year in twentieth century",
-         "input": 1998,
+         "description": "year not divisible by 4: standard year",
+         "input": 2015,
          "expected": false
       },
       {
-         "description": "standard year in nineteenth century",
-         "input": 1900,
-         "expected": false
+         "description": "year divisible by 4, not divisible by 100: leap year",
```

Comment:
No worries - its good to make well thought decisions - especially when aiming at relatively inexperienced people - and in the end many future tests will draw from the templates in x-common.
How do I proceed from now on? I intend to squash all my commits, thereby amending the commit description. Should I wait on @stevejb71 ?